DSharpPlus/Net/DiscordApiClient.cs

```diff
@@ -774,17 +774,7 @@ internal async Task<DiscordMessage> CreateMessageAsync(ulong channel_id, string
 
         internal async Task<DiscordMessage> CreateMessageAsync(ulong channel_id, DiscordMessageBuilder builder)
         {
-            if (builder.Content != null && builder.Content.Length > 2000)
```

Comment:
You remove these checks here but don't specify them in the builder validate method. I would copy these checks and place them in the validate `else` statement (where it doesn't modify).

Reply:
The Content Prop of the message builder already does this check for line 777.